context.go

```diff
@@ -321,6 +322,64 @@ func (c *Context) Header(key, value string) {
 	}
 }
 
+func (c *Context) SetCookie(name string, value string, options ...interface{}) {
+	cookie := http.Cookie{}
+	cookie.Name = name
+	cookie.Value = url.QueryEscape(value)
+
+	if len(options) > 0 {
```

Comment:
This sounds risky. Why just add the max age parameter in the SetCookie signature?